package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "mocha",
+    "test-watch": "mocha --watch",
```

Comment:
I added this script in here as well to make it easier to make fixes and test at the same time.